src/classes/Country.cls

```diff
@@ -131,4 +129,38 @@ public with sharing class Country {
         
         System.debug(countryWithCodeMap.get(countrycode));
     }
-}
\ No newline at end of file
+
+    /**   Description :   Method Gettting the Country with letter and Greater District Form Integer
+    *     Created By  :   Sagar Nirwal
+    *     Arguments   :   String,Integer
+    *     Return      :   void
+    *     Revision log:   V_1.0 - Created  - Youe name - Date
+    *                     V_1.1 - Modified - Youe Name - Date - Details what you modified 
+    **/
+    public static void combinationCountryStateDistrict(String combinationLetter,Integer districtSize){
+        
+        for(String countryName : countryMap.keySet()){
+            if(countryName.startsWith(combinationLetter)){
+                for(String stateName : countryMap.get(countryName).keySet()){
+                    if(stateName.startswith(combinationLetter)){
+                        for(String districtName : countryMap.get(countryName).get(stateName) ){
+                            if(districtName.startsWith(combinationLetter)){
+                                System.debug('Combination - Country : '+countryName+', State : '+stateName+ ', District : '+districtName);
+                            }
+                        }
+                    }
+                }    
+            }
+        }
+        System.debug('District with Greater Interger : '); 
+        for(String countryName : countryMap.keySet()){
+            for(String stateName : countryMap.get(countryName).keySet()){
+               	for(String districtName : countryMap.get(countryName).get(stateName) ){
+                    if( districtName.length() >= districtSize){
+                        System.debug(districtName);
+                    }
```

Comment:
Add one more if here and check if a country, state, or district name starts with a combination letter.
If yes then use the following debug.
System.debug('Combination - Country : '+countryName+', State : '+stateName+ ', District : '+districtName);
And please remove the code from 142 to 154.